src/resources/postcss/components/_top-bar.pcss

```diff
@@ -0,0 +1,61 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Top Bar
+ *
+ * ----------------------------------------------------------------------------- */
+.tribe-events {
+	.tribe-events-calendar-top-bar {
+		display: flex;
+		margin-top: var(--spacer-7);
+		margin-bottom: var(--spacer-5);
+		flex-direction: row;
+		flex-wrap: wrap;
+		align-items: center;
+	}
+
+	.tribe-events-calendar-top-bar--nav {
+		display: flex;
+		padding-right: 15px;
+		align-items: center;
+
+		nav a {
```

Comment:
We should target all elements with classes rather than nesting like so. Every element should have a unique class name using BEM format so that we should really only be using one class.

For example, we may have `.tribe-events__top-bar-nav-wrapper`, `.tribe-events__top-bar-nav`, and `.tribe-events__top-bar-nav-link`, for example.